### PR TITLE
Docs 4812 visualizer april30 vh

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

**Review by asimons-mule — COMMENTED** [review]
Looks great!! A few thoughts...

1) There is an extra + sign after 4.1.1 in supported versions
2) I like the simplification for the known limitations, but there are some important differences between Mule 3 and Mule 4 that are missing. Perhaps they don't need to be separate bullets, but we may want to call out the differences. For example, in Mule 4 the issue persists even with the new visualizer agent. We also don't reference "old" and "new" agents in the docs, but rather by their dates (Mule 3 = April 5). 
3) I think we're missing known limitation #5 from my notes.
4) "Set the following property in the server's properties" line didn't get removed. For RTF the property is not set at the application level.
5) In the reduce performance impact section, may we please document what the header injection property is? It got removed at some point a few versions back.. 
6) What is MuleSoft Documentation Contributor’s Agreement????? ;)

**Review by asimons-mule — COMMENTED** [review]
Looks awesome! Final comment is that anypoint.platform.config.analytics.agent.disabled=true shouldn't be removed from Reduce Potential Perf Page. Header injection property should just be added in addition.

File: modules/ROOT/pages/setup.adoc [hunk 1]
```diff
@@ -53,42 +53,72 @@ To enable the Anypoint Monitoring agent:
...
 
+=== Runtime Fabric Setup
+
+Follow the instructions for xref:runtime-fabric::install-create-rtf-arm.adoc[setting up Runtime Fabric]. After you install Runtime Fabric, Visualizer can display applications that are running and deployed to Runtime Fabric.
+
+Visualizer relies on a header injection to display connections between services correctly. The header injection is disabled by default for Mule 4. The correct connections will not be displayed for your Mule 4 Runtime Fabric applications unless you <<enable_header_injection,enable the header injection>>. 
+
+Known Limitations:
+
+* If a Mule 3 CloudHub application with the old Visualizer agent (version earlier than April 5, 2019) is calling a Runtime Fabric ingress endpoint, there will be extra edge from the CloudHub app to a node with the IP address for the controller. Please upgrade to the latest release of the Mule runtime engine.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/setup.adoc]: "calls" instead of "is calling" (global) 
 "an extra edge"

File: modules/ROOT/pages/setup.adoc [hunk 2]
```diff
@@ -53,42 +53,72 @@ To enable the Anypoint Monitoring agent:
...
+* If a Mule 4 CloudHub application is calling a Runtime Fabric ingress endpoint, there will be and extra edge from the CloudHub app to a node with the IP address for the controller. You can use your filters within Visualizer to hide this node.
+* If a Runtime Fabric Mule app is calling a Mule 3 CloudHub Mule app with the old Visualizer agent (version earlier than April 5, 2019) using its DNS, the correct connection will be drawn, but there will be an additional edge from the *External Traffic* node to the CloudHub application.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/setup.adoc]: global: the correct connection is drawn (helps with the contrast for "there will be an additional edge" which is probably best in passive construction).

File: modules/ROOT/pages/setup.adoc [hunk 3]
```diff
@@ -53,42 +53,72 @@ To enable the Anypoint Monitoring agent:
...
+* If a Runtime Fabric Mule app is calling a Mule 4 CloudHub Mule app using its DNS, the correct connection will be drawn, but there will be an additional edge from the *External Traffic* node to the CloudHub application.
+* If you have an application in two different clusters with the same name, these applications will appear as one node on the canvas. You should rename one application.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/setup.adoc]: these applications are drawn as one node (to get rid of "will" and "appears")?

File: modules/ROOT/pages/setup.adoc [hunk 4]
```diff
@@ -53,42 +53,72 @@ To enable the Anypoint Monitoring agent:
...
+* If you have an application in two different clusters with the same name, these applications will appear as one node on the canvas. You should rename one application. 
+
+
+[[enable_header_injection]]
 === Enable Header Injection
 
-Visualizer requires an additional header injection to work properly.
-However, due to a potential performance impact for Mule 4.1.x, header injection has been disabled by default.
-You must enable it at the server, not application, level.
+Visualizer requires an additional header injection to work properly for standalone Mule and Runtime Fabric deployments.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/setup.adoc]: you can delete "to work properly"?

**Review by Mulesoft-Mysti — APPROVED** [review]
One missing word, everything else is not a showstopper.

**Review by asimons-mule — APPROVED** [review]
Looks good to me!

**Pull request merged**